doc/release/1.6.0-notes.rst

```diff
@@ -17,9 +17,9 @@ run your code with ``python -Wd`` and check for ``DeprecationWarning`` s).
 Our development attention will now shift to bug-fix releases on the
 1.3.x branch, and on adding new features on the master branch.
 
-This release requires Python 3.6+ and NumPy 1.14.5 or greater.
+This release requires Python 3.6+ and NumPy 1.16.5 or greater.
```

Comment:
I guess 3.7+ now too, although maybe we can just keep this MR focused on its original purpose.

Reply:
yeah I considered that and left it out on purpose, to not do two things at once.